admin/app/adapters/organization.js

```diff
@@ -18,8 +16,11 @@ export default class OrganizationAdapter extends ApplicationAdapter {
     url = this.urlPrefix(url, this.buildURL(snapshot.modelName, snapshot.id, null, 'findHasMany'));
 
     if (relationship.type === 'organization-membership' && snapshot.adapterOptions) {
-      const options = queryString.stringify(snapshot.adapterOptions);
-      url = `${this.host}/${this.namespace}/organizations/${snapshot.id}/memberships?${options}`;
+      const params = new URLSearchParams();
+      for (const [key, value] of Object.entries(snapshot.adapterOptions)) {
+        if (value) params.append(key, value);
+      }
```

Comment:
Le constructeur peut prendre directement du clé/valeur

```suggestion
      const params = new URLSearchParams(snapshot.adapterOptions);
```

Reply:
Oui c'est ce que j'avais fait dans un premier temps, mais les valeurs `null` sont sérialisées par `URLSearchParams` puis interprétées côté serveur en chaîne "null". J'ai donc fait en sorte de ne pas les envoyer comme c'est fait dans la lib `query-string`